conviva/src/main/java/com/bitmovin/analytics/conviva/ConvivaAnalyticsIntegration.java

```diff
@@ -600,15 +600,15 @@ private synchronized void transitionState(ConvivaSdkConstants.PlayerState state)
     };
 
     private final EventListener<PlayerEvent.Error> onPlayerErrorListener = event -> {
-        Log.d(TAG, "[Player Event] Error");
+        Log.d(TAG, "[Player Event] Error - " + event.getCode().getValue());
```

Comment:
Mini nit: extract the code into a variable.

Reply:
[42bc670](https://github.com/bitmovin/bitmovin-player-android-analytics-conviva/pull/86/commits/42bc670b2fbd29e6800c8fc7e6b9b38f40f4b14d)